ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py

```diff
@@ -75,15 +75,15 @@
         "volumes",
         "volume_collections"
       - NOTE:-
```

Comment:
I see this convention ":- " used in some places, can we just use ": "?

Reply:
not possible.. ':' gives rendering error for ansible-doc command